anmykh/homework/homework_8_chefs.py

```diff
@@ -0,0 +1,30 @@
+class AbstractCook:
+    food_name = "Food"
+    drink_name = "Drink"
+    food_total = 0
+    drink_total = 0
+
+    def add_food(self, count: int, price: int):
+        self.food_total += count*price
```

Comment:
spaces around operator

Reply:
if I add 1 more space - it says "Too many blank lines"
<img width="874" alt="Screenshot 2024-02-23 at 17 43 01" src="https://github.com/kudinov-fedor/python-core-training/assets/107838250/1d112133-20db-465f-b8e4-ee660bdd87eb">

Reply:
That`s why I didn`t add it

Reply:
I mean around operator


self.food_total += count*price

self.food_total += count * price